catroid/src/main/AndroidManifest.xml

```diff
@@ -209,6 +209,26 @@
             </intent-filter>
         </activity>
         <activity
+            android:name=".ui.AccessibilityPreferencesActivity"
+            android:label="@string/preference_access_title_activity_access"
+            android:noHistory="false"
+            android:screenOrientation="portrait">
+            <intent-filter>
+                <action android:name=".ui.AccessibilityPreferencesActivity" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
```

Comment:
Remove intent filter. AccessibilityPreferencesActivity should not be startable by other apps (e.g. the effect is that a dialog appears if you click on "Accessibility" in the preferences and another flavor of Pocket Code is installed, e.g. a standalone apk or the new create@school version).